source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRViewController.mm

```diff
@@ -58,7 +58,43 @@ - (void)render
     UIView *view = self.view;
     view.frame = _guideFrame;
     NSMutableArray *inputs = [[NSMutableArray alloc] init];
-
+    
+    std::string backgroundImage = _adaptiveCard->GetBackgroundImage();
+    NSString* imgUrl = nil;
+    if(backgroundImage.size())
```

Comment:
.size() is an int. If you want to check for an empty string, use the .empty() method.